provisions/roles/cleanup/jenkins/master/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Stop jenkins systemd service
+  sudo: yes
+  service: name=jenkins state=stopped
+
+- name: Remove contents under direcotry /var/lib/jenkins/
+  sudo: yes
+  shell: rm -rf /var/lib/jenkins/*
```

Comment:
@bamachrn : Okay. But then if there are code changes related to jenkins and scripts which reside in `/var/lib/jenkins/*` directory, the deployment dev needs to manually clear it, since it is kept as cache and just a normal deployment does not update scripts inside /var/lib/jenkins directory.